pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 79
```

Comment:
Before you hit merge: the `black` default is 88. Which is probably what got used in #161 

I think if you aren't as stuck in your ways as I am 88 is a good choice (and simplifies things).

Reply:
oh... you know i had thought i reran it using 79 but you are likely right @betatim 
@mbjoseph do you have thoughts on 79 vs 88 character line width? pycharm defaults to 88 with black. i'm still trying to figure out how to fix that. should we stick to pep8 convention? black actually does break a few things like the single vs double quote rules already. thoughts??

Reply:
ok i did fix this to be 79 last night @betatim @mbjoseph  -- in my last commit [here](564a8a83895ee53ac60c93cd402954bb416e9758)

we are in agreement that we'd like to use 79 for this repo (just chatted with max) SO... can we merge this?! i'm excited to have everyone apply this on their pr's and such too.